### PR TITLE
DataStructures: add more examples of maps

### PR DESCRIPTION
**Pull request by poko91**

### CODE REVIEW

**Review by lomkju — APPROVED** [review]
LGTM

File: GolangPractice-02-DataStructures/maps.go [hunk 1]
```diff
@@ -3,14 +3,21 @@ package main
 import "fmt"
 
 func main() {
-	// To create an empty map use syntax - make(map[key-type]val-type)
```
Comment (lomkju) [inline GolangPractice-02-DataStructures/maps.go:6]: Why did you remove this comment?

File: GolangPractice-02-DataStructures/maps.go [hunk 2]
```diff
@@ -3,14 +3,21 @@ package main
...
+	// Declare an empty map
+	var person = map[string]int{}
+	fmt.Println("1. Empty map created having string as key-type and int as value-type:")
+	fmt.Println(person)
+
+	// Map declaration using make function
 	m := make(map[string]int)
 
-	// To add more key-value pairs to the map use syntax: m[key] = value
+	// Adding key-value pairs to the map
```
Comment (lomkju) [inline GolangPractice-02-DataStructures/maps.go:15]: I like the previous comment why don't you use that?

**Review by lomkju — CHANGES_REQUESTED** [review]

File: GolangPractice-02-DataStructures/maps.go (hunk 1 above)
Comment (poko91) [reply GolangPractice-02-DataStructures/maps.go:6]: The comment was wrong for the syntax used, so I have deleted the comment.

**Review by lomkju — APPROVED** [review]
